microsoft-edge/webview2/reference/win32/0-9-538/icorewebview2newwindowrequestedeventargs.md

```diff
@@ -47,6 +47,8 @@ IsUserInitiated is true when the new window request was initiated through a user
 
 > public HRESULT [get_IsUserInitiated](#get_isuserinitiated)(BOOL * isUserInitiated)
 
+The Edge popup blocker is disabled for WebView so the app can use this flag to block non-user initiated popups.
```

Comment:
```suggestion
WebView2 controls may display pop-up windows because the Pop-Up blocker is turned off. To block non-user initiated pop-up windows from being displayed, use `get_IsUserInitiated`.
```